### ISSUE TITLE
find drivers in close to a user

### ISSUE THREAD

**Issue by danceasarxx** [danceasarxx/transport-dev#5]

**Comment by danceasarxx** [comment]
Users need to find drivers close to them and also able to carry the load they want to carry.  Hence the client  must send the user's location and the type of driver he needs

**Comment by danceasarxx** [comment]
The server would need to talk to google maps api I think so as to know which drivers are close by

**Comment by phackHaulage** [comment]
Yes agreed then the user needs to communicate to the user the best results about 5 and have an option for see more.

{driverName, ratingof the driver, id of the driver}

**Comment by danceasarxx** [comment]
The user does not need the id of the driver

**Comment by phackHaulage** [comment]
the comment has been updated to the client

**Comment by phackHaulage** [comment]
the reserve by the client should send the id of the selected driver to the server and the contact information should be sent back to the user for the user to contact the driver
